passes/sat/freduce.cc

```diff
@@ -27,6 +27,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <algorithm>
+#include <limits>
```

Comment:
Is this include needed?

Reply:
Yes it is, gcc version in GH-actions changed (example failure in dependabot PR: https://github.com/antmicro/yosys/pull/320/checks?check_run_id=2601015215#step:5:1471), and now limits needs to be include explicite.

It is already fixed on mainstream yosys.